### ISSUE TITLE
Material3 OutlinedTextField outline is drawn behind label during screenshot test

### ISSUE THREAD

**Issue by kikin81** [cashapp/paparazzi#1032]
**Description**
When taking a paparazzi screenshot of a Material3 `OutlinedTextField` you can see the outline drawn behind the label text. This is not present in the Android Studio compose preview.

**Steps to Reproduce**
Example paparazzi test:

```Kotlin
class LaunchViewTest {
    @get:Rule
    val paparazzi = Paparazzi(
        deviceConfig = PIXEL_5,
        theme = "android:Theme.Material.Light.NoActionBar"
    )

    @Test
    fun launchComposable() {
        paparazzi.snapshot {
            Column(
                modifier = Modifier
                    .fillMaxWidth()
                    .background(color = Color.White)
                    .padding(16.dp)
            ) {
                OutlinedTextField(
                    label = {
                        Text(text = "Example of a label")
                    },
                    value = "user input string",
                    onValueChange = { },
                    modifier = Modifier.fillMaxWidth()
                )
            }
        }
    }
}
```

**Expected behavior**
The paparazzi screenshot test should match the Android Studio preview which does not show the outline behind the label

**Additional information:**
- Paparazzi Version: 1.3.1 (also in 1.2.0)
- OS: Android 13
- Compile SDK: 34
- Gradle Version:  8.2
- Android Gradle Plugin Version: 8.0.2

**Screenshots**

| Expected result (AS Preview)  | Actual result (paparazzi screenshot) |
| ------------- | ------------- |
| ![Screenshot 2023-08-01 at 3 01 00 PM](https://github.com/cashapp/paparazzi/assets/834204/209ca2cc-f4dc-4a0a-bc0e-46f22ed40e3e) | ![us kikin android paparazzitest_LaunchViewTest_launchComposable](https://github.com/cashapp/paparazzi/assets/834204/81413332-35af-4d88-9cd0-579bf5bea427) |

**Comment by BrianGardnerAtl** [comment]
Hey! I checked in on this and it seems to be an animation issue. The border is created by the `BorderBox` composable, where the border stroke is created from the `animateBorderStrokeAsState` function. I actually found this through preview where I noticed if you change the value you will see the non-cutout border for one frame before it settles. Running the test with gif shows that only the first frame has the border and the cutout is applied afterwards.

I filed [an issue on the compose bug tracker](https://issuetracker.google.com/issues/322314695) to see if this is intentional and also filed a [feature request for paparazzi](https://github.com/cashapp/paparazzi/issues/1261) to provide a way to snapshot after these types of animations.

I don't think this is something we can solve right now but using the `gif` api can show the border cut out for this component.
